common/persistence/sql/common.go

```diff
@@ -58,7 +62,9 @@ func (m *sqlStore) txExecute(operation string, f func(tx sqldb.Tx) error) error
 	}
 	err = f(tx)
 	if err != nil {
-		tx.Rollback()
+		if err := tx.Rollback(); err != nil {
+			err = errors.Wrap(err, "failed to rollback transaction")
```

Comment:
why do we need to wrap it?

Reply:
We don't just updated it to just swap the error.